test/NuGet.Core.FuncTests/Dotnet.Integration.Test/DotnetRestoreTests.cs

```diff
@@ -1500,6 +1512,38 @@ public void Dotnet_New_Template_Restore_Success(string template)
             }
         }
 
+        [PlatformTheory(Platform.Linux)]
+        [InlineData("nunit-test")]
+        [InlineData("razorcomponent")]
+        [InlineData("page")]
+        [InlineData("viewimports")]
+        [InlineData("viewstart")]
+        [InlineData("gitignore")]
+        [InlineData("globaljson")]
+        [InlineData("nugetconfig")]
+        [InlineData("tool-manifest")]
+        [InlineData("webconfig")]
+        [InlineData("sln")]
+        [InlineData("proto")]
+        public void Dotnet_New_Template_CreateItem_Success(string template)
+        {
+            // Arrange
+            using (var pathContext = new SimpleTestPathContext())
+            {
+                var projectName = new DirectoryInfo(pathContext.SolutionRoot).Name;
+                var solutionDirectory = pathContext.SolutionRoot;
+
+                // Act
+                CommandRunnerResult newResult = _msbuildFixture.RunDotnet(solutionDirectory, "new " + template);
+
+                // Assert
+                // Make sure new template action was success.
+                newResult.Success.Should().BeTrue(because: newResult.AllOutput);
+                // No actual restore happen here
```

Comment:
Not sure if I need to create new `.cs` file to place tests doesn't actually do restore, but feels unnecessary.